cli/mobycli/scout_suggest.go

```diff
@@ -0,0 +1,73 @@
+/*
+   Copyright 2023 Docker Compose CLI authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package mobycli
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/docker/compose/v2/pkg/utils"
+
+	"github.com/fatih/color"
+)
+
+func displayScoutQuickViewSuggestMsgOnPull(args []string) {
+	image := pulledImageFromArgs(args)
+	displayScoutQuickViewSuggestMsg(image)
+}
+
+func displayScoutQuickViewSuggestMsgOnBuild(args []string) {
+	// only display the hint in the main case, build command and not buildx build, no output flag, no progress flag, no push flag
+	if utils.StringContains(args, "--output") || utils.StringContains(args, "-o") ||
+		utils.StringContains(args, "--progress") ||
```

Comment:
Ah I see `--progress` but not `BUILDKIT_PROGRESS` env var being taken into account though.

Reply:
I think we should just check `--progress` in `metrics.HasQuietFlag`.

Reply:
I'll add the env var 👍